Sources/OpenAPIAsyncHTTPClient/AsyncHTTPClientTransport.swift

```diff
@@ -68,14 +68,22 @@ public struct AsyncHTTPClientTransport: ClientTransport {
         /// The HTTP client used for performing HTTP calls.
         public var client: HTTPClient
 
+        /// The default shared HTTP client.
+        ///
+        /// This is a workaround for the lack of a shared client
+        /// in AsyncHTTPClient. Do not use this value directly, outside of
+        /// the `Configuration.init(client:timeout:)` initializer, as it will
+        /// likely be removed in the future.
+        public static let sharedClient: HTTPClient = .init()
+
         /// The default request timeout.
         public var timeout: TimeAmount
 
         /// Creates a new configuration with the specified client and timeout.
         /// - Parameters:
         ///   - client: The underlying client used to perform HTTP operations.
         ///   - timeout: The request timeout, defaults to 1 minute.
-        public init(client: HTTPClient = .init(), timeout: TimeAmount = .minutes(1)) {
+        public init(client: HTTPClient = Self.sharedClient, timeout: TimeAmount = .minutes(1)) {
```

Comment:
Instead of making the shared client public you can instead make the client argument optional, default it to nil and use the the internal shared client if the parameter passed in is nil

Reply:
Oh that's better, yeah

Reply:
@dnadoba Updated, can you take a look?